technical_documentation/ci_cd_approach.md

```diff
@@ -1 +1,9 @@
-TODO
\ No newline at end of file
+# Continuous Integration and Continous Delivery
+The project environment consists of multiple loosely coupled, independet microservices. This characteristic requires strict and disciplined orchestration of the whole context.
+Therefore, the following automation mechanisms were introduced in terms of satisfying the CI/CD idea, to ease the deployment process and provide the latest features directly to the end-users:
+* To ease the deployment and dependency management, all microservices are managed in an Docker container
+* These docker images are linked to a Jenkins pipeline, which takes care of the continuous deployment on a third party server. Additionally, the
```

Comment:
Könntest hier noch erwähnen, dass automatisch bei jedem Push in den Master Branch automatisch neu deployed wird.

Reply:
Eventuell könnte man hier noch erwähnen, dass wir grundsätzlich immer PR erstellt haben und nocheinmal jemand gereviewed hat, bevor wir etwas in den Master Branch mergen. Weiß nur nicht, ob das hier die richtig hinpasst, aber irgendwo könnten wir das erwähnen, da wir es ja schon Konsequent durchgezogen haben und irgendwo auch die Qualität des finalen Codes der im Master-Branch ist und deployed wird verbessert. Was meinst du?

Reply:
hab es ergänzgt. das waren gute punkte